src/image/VisitInfo.cc

```diff
@@ -320,7 +320,9 @@ class VisitInfoFactory : public table::io::PersistableFactory {
         int version = getVersion(record);
         if (version > SERIALIZATION_VERSION) {
             throw LSST_EXCEPT(pex::exceptions::TypeError, "Cannot read VisitInfo FITS version > " +
-                                                                  std::to_string(SERIALIZATION_VERSION));
+                                                                  std::to_string(SERIALIZATION_VERSION) +
+                                                                  ", found version " +
+                                                                  std::to_string(version) + " in file.");
```

Comment:
Ending the log message in "in file. " is unnecessary and potentially confusing. This could lead a user to expect a path to appear after that, or think that the path did not appear for whatever reason.